dc-cudami-server/dc-cudami-server-webapp/src/main/java/de/digitalcollections/cudami/server/controller/v1/identifiable/entity/parts/V1WebpageController.java

```diff
@@ -151,9 +177,12 @@ private void convertLocalizedTextXml(Element xml) {
     xml.setContent(translations);
   }
 
-  private Element createDocumentsElement(Element content) {
+  private Element createDocumentsElement(List<Element> contents) {
     Element documents = new Element("documents");
-    documents.setContent(content.clone());
+    contents.forEach(
+        (content) -> {
```

Comment:
Glaube hier sind Streams echt overkill, ein `for Element content : contents` wäre lesbarer. Ditto für den Rest der `.forEach` calls in dem PR. Würde `.forEach` nur dann einem for-each Loop vorziehen wenn die Funktion wirklich winzig ist bzw. nur eine Methodenreferenz, oder am Ende einer Stream-Kette steht die vorher filtert/mappt.

Reply:
Erledigt :+1: